nixos/modules/services/misc/ombi.nix

```diff
@@ -0,0 +1,79 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let cfg = config.services.ombi;
+
+in {
+  options = {
+    services.ombi = {
+      enable = mkEnableOption ''
+        Ombi.
+        Optionally see https://docs.ombi.app/info/reverse-proxy/ on how to set up a reverse proxy.
```

Comment:
You don't need the final `.` with `mkEnableOption`.

Might also be nice to use `<xlink` around the link so it is easily clickable from docs.